modules/stochastic_tools/src/samplers/MonteCarloSampler.C

```diff
@@ -11,10 +11,6 @@
 #include "Distribution.h"
 
 registerMooseObjectAliased("StochasticToolsApp", MonteCarloSampler, "MonteCarlo");
```

Comment:
this isnt recognized by the docs it seems

Reply:
This seems to be used all over the place though...

Reply:
fixed it